<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeSettings</class>
 <widget class="QDialog" name="TradeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交易设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>券商佣金：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spbBrokerFeePercentage">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="prefix">
        <string>万</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>最低</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbMinBrokerFee">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="suffix">
        <string>元</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>印花税：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spbStampDuty">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string>‰</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>TradeSettings</receiver>
   <slot>save_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save_settings()</slot>
 </slots>
</ui>
